lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3263,7 +3281,20 @@ swift::ModuleDecl *SwiftASTContext::GetModule(const SourceModule &module,
   auto import_diags = getScopedDiagnosticConsumer();
 
   // Perform the import.
```

Comment:
nit: Technically the import happens in the `ast->getModuleByName` call, so I'd move the progress setup above the comment.

Reply:
Good point, I'll change this.